questionpy_server/worker/runtime/package.py

```diff
@@ -85,8 +85,8 @@ def get_path(self, path: str) -> Traversable:
 
     def setup_imports(self) -> None:
         for new_path in (
-            str(self.path / f"{DIST_DIR}/dependencies/site-packages"),
-            str(self.path / f"{DIST_DIR}/python"),
+            str(self.path / DIST_DIR / "dependencies/site-packages"),
+            str(self.path / DIST_DIR / "python"),
```

Comment:
Das würde auf Windows seltsame Pfade zurückgeben, oder?
z.B. `C:\path\to\file.zip\dist\dependencies/site-packages`

Die Konvention ist ja bei `sys.path` auf allen Plattformen nur forward slashes zu verwenden.

Also, vielleicht so?
`(self.path / DIST_DIR / "dependencies" / "site-packages").as_posix()`

Beispiel:

```py
path = Path("/path/to/file.zip")
import_path = (path / "dist" / "dependencies" / "site-packages").as_posix()
print(f"import_path = {import_path}")

win_path = PureWindowsPath(r"c:\path\to\file.zip")
win_import_path = (win_path / "dist" / "dependencies" / "site-packages").as_posix()
print(f"win_import_path = {win_import_path}")
```

```
import_path = /path/to/file.zip/dist/dependencies/site-packages
win_import_path = c:/path/to/file.zip/dist/dependencies/site-packages
```

Reply:
Ist mir an sich ziemlich egal :shrug: 

Wegen `as_posix()` - ich habe gerade kein Windows zur hand, aber ich nehme an, dass Python dort das native Format erwartet, nicht `/`?